specification/network/resource-manager/readme.md

```diff
@@ -30,13 +30,17 @@ description: Network Client
 openapi-type: arm
 tag: package-2024-01
```

Comment:
This tag doesn't exist anymore. Can you update it to `package-2024-01-preview` either?

Reply:
changing the tag is causing Avocado errors stating we have [MISSING_APIS_IN_DEFAULT_TAG](https://github.com/Azure/avocado/blob/master/README.md#MISSING_APIS_IN_DEFAULT_TAG)

Reply:
package-2024-01 is the latest stable, so I don't think that's the issue reverting back to package-2024-01. example PR for when we merged an old preview version https://github.com/Azure/azure-rest-api-specs/commit/18ebda844331bc4b52babc60bd27112e4143f66c#diff-06b33c899bdfd2b8060eff3b065641a8f1b0ac7a8188d896d9cc2965ddfb22fe

Reply:
The problem is `package-2024-01` tag definition doesn't exist in the readme.md that cause the input file cannot be found while generating the SDK.